handlers/label_watchers/tests/not_watched.json

```diff
@@ -18,7 +18,10 @@
     }, 
     "action": "labeled", 
     "issue": {
-      "number": 84
+      "number": 84,
+      "user": {
+        "login": "jdm"
```

Comment:
This should be a different user to ensure the test is testing what we want, I think.